packages/kbn-expandable-flyout/src/index.stories.tsx

```diff
@@ -118,13 +122,17 @@ export const Right: Story<void> = () => {
 
 export const Left: Story<void> = () => {
   const state = {
-    right: {
-      id: 'right',
-    },
-    left: {
-      id: 'left',
+    byId: {
+      memory: {
+        right: {
+          id: 'right',
+        },
+        left: {
+          id: 'left',
+        },
+        preview: [],
+      },
     },
-    preview: [],
   } as unknown as State;
```

Comment:
should remove these `as uknown as` that are present throughout the tests, or else this is highly likely to happen again in the future